.github/workflows/_build.yaml

```diff
@@ -140,3 +140,14 @@ jobs:
           ${{ env.PACKAGE_PATH }}/bin/bashparser
```

Comment:
After our discussion, we decided to remove these artifacts as they will be part of the wheel anyway?

Reply:
You can see my answer to this issue here - https://github.com/oracle-samples/macaron/pull/76#discussion_r1137896832

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/d2fe70dc8bc12b52071d18e885bf63b86f9dcc25